env0/resource_aws_credentials.go

```diff
@@ -27,7 +30,6 @@ func resourceAwsCredentials() *schema.Resource {
 				Optional:      true,
 				ForceNew:      true,
 				ConflictsWith: []string{"access_key_id"},
-				ExactlyOneOf:  []string{"access_key_id"},
```

Comment:
The "ExactlyOneOf" cannot be inforced in the schema level. (Will fail on import).